VAMobile/src/constants/analytics.ts

```diff
@@ -572,11 +572,20 @@ export const Events = {
       },
     }
   },
-  vama_login_start: (isSIS = false): Event => {
+  vama_login_readme: (isSIS = false): Event => {
```

Comment:
SIS is the only available login method now so I don't think this is required anymore and everything should be defaulted to true now

Reply:
I decided to remove sis since I though it may not be necessary for the readme event